sycl/test-e2e/Graph/Inputs/kernel_bundle_spirv.cpp

```diff
@@ -0,0 +1,98 @@
+// Tests creating a node using a SPIR-V kernel imported with
+// sycl_ext_oneapi_kernel_compiler_spirv. The SPIR-V kernels used in this test
+// are identical to the ones used in KernelCompiler/Kernels/kernels.spv
```

Comment:
I struggled to generate spir-v for level-zero. Tried with SYCL_DUMP_IMAGES but the driver doesn't accept it. 
So I'm reusing a kernel from another test for the sycl_ext_oneapi_kernel_compiler_spirv extension.

Would have to do further debugging to find the correct way to generate / write spir-v for level-zero. But I think this kernel works well enough for what we are trying to test here.

Reply:
I think that's fine, if required we can investigate more in the future. Perhaps getting in contact with the original author of those files or other folks might help.

Reply:
Agree that this is fine for now.